### DISCUSSION TITLE
Odd Data Names v3.5.0 Only One Machine

### DISCUSSION THREAD

**Discussion by gary-hammer** [Daveiano/weewx-wdc#257] [Q&A]
I have installed 3.5.0 on two machines using the recommended method of uninstalling the old version, wiping the output directories, installing the new.
My install on WeeWX v5.02 using the pip install method is working and displaying fine.
My install on WeeWX 4.10 using setup.py method is displaying two odd names for values. Apparent Temperature and Cloud Base seem to show a variable name instead of the wanted display name.
My question is, where does WDC get the names for the tiles?

See attached screenshots for what I mean.
<img width="871" alt="wdc-350-v5" src="https://github.com/Daveiano/weewx-wdc/assets/12861605/d969f312-51b2-4e94-9efa-9d80e2db4a4c">
<img width="869" alt="wdc-350-v4" src="https://github.com/Daveiano/weewx-wdc/assets/12861605/98e220ec-7bc8-4ad4-b3ea-50f7c84b68b5">

**Comment by Daveiano** [comment]
The observation labels should come from the lang files, here: https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/lang/en.conf

Perhaps there is something configured in `weewx.conf`? Like here: https://github.com/Daveiano/weewx-wdc/wiki/WeeWX-related#changing-the-label-of-an-observation-for-the-whole-skin

**Reply by gary-hammer** [reply]
Thanks. I had searched those files before but found nothing. Not to say the offending entry wasn't there, just that I didn't find them.

I took the 'easy' way out.
I built a new server and installed everything fresh. Probably a good thing as the old server had been there since weewx v3 and had many things poked in and out of it.

No issues now.

**Reply by Daveiano** [reply]
Great news, thank you!
